manim/mobject/types/vectorized_mobject.py

```diff
@@ -458,13 +477,27 @@ def start_new_path(self, point):
         self.append_points([point])
         return self
 
-    def add_cubic_bezier_curve(self, anchor1, handle1, handle2, anchor2):
+    def add_cubic_bezier_curve(
+        self, anchor1: np.ndarray, handle1: np.ndarray, handle2: np.ndarray, anchor2
```

Comment:
can you add an example here?
e.g. this one 
```py
class BezierSpline(Scene):
    def construct(self):
        np.random.seed(42)
        area = 4
        p1 = np.array([-3, 1, 0])
        p1b = p1 + [1, 0, 0]
        d1 = Dot(point=p1).set_color(BLUE)
        l1 = Line(p1, p1b)
        self.add(l1, d1)
        p2 = np.array([3, -1, 0])
        p2b = p2 - [1, 0, 0]
        d2 = Dot(point=p2).set_color(RED)
        l2 = Line(p2, p2b)
        self.add(l2, d2)
        l1.add_cubic_bezier_curve(p1b, p1b + 3 * RIGHT, p2b - 3 * RIGHT, p2b)
```
![image](https://user-images.githubusercontent.com/44469195/105748314-454bae00-5f42-11eb-80f2-412c93b93bc0.png)

Reply:
Hmmmm do you think this is meant to be used by the end-user?

Reply:
good question.
I think the possibility to make a bezier spline would be a nice and useful feature. But I am not sure if it should be used like this.

Reply:
Yes, I'm not sure too. For me, this is mean to be used for an internal use only.

Reply:
okay, then we leave it without an example

Reply:
Additonnaly, there is already `CubicBezier` for that IIRC